driver-core/src/main/java/com/datastax/driver/core/SchemaParser.java

```diff
@@ -663,6 +663,24 @@ else if (targetType == AGGREGATE)
             return whereClause;
         }
 
+        // Used by maybeSortUdts to sort at each dependency group alphabetically.
+        private static final Comparator<Row> sortByTypeName = new Comparator<Row>() {
+            @Override
+            public int compare(Row o1, Row o2) {
+                String type1 = o1.getString(UserType.TYPE_NAME);
+                String type2 = o2.getString(UserType.TYPE_NAME);
+
+                if (type1 == null && type2 == null) {
```

Comment:
Technically these should never be null, could remove check if you think it is being overcautious